coral/README.md

```diff
@@ -24,10 +24,22 @@
 
 ℹ️ Coral uses `pnpm run` as a package manager. Read their official documentation [how to install](https://pnpm.io/installation) it.
 
+### Usage: How to run Coral inside the Klaw application
+
+1. navigate to this directory
+2. run `pnpm install`
+3. run `make enable-coral-in-springboot` (see our [Makefile](Makefile)). This will enable Coral in Klaw. It also add the current Coral build files in the right place.
```

Comment:
```suggestion
3. run `make enable-coral-in-springboot` (see our [Makefile](Makefile)). This will enable Coral in Klaw. It also moves the Coral build files to the right directory.
```